CMakeLists.txt

```diff
@@ -51,18 +45,31 @@ file(GLOB ERROR_SRC
     "error/*.c"
 )
 
-# The SIKE code #includes .c files directly, including all sike_r*/*.c breaks the build due to duplicates
-file(GLOB PQ_SRC
-        "pq-crypto/*.c"
-        "pq-crypto/bike_r1/*.c"
-        "pq-crypto/bike_r2/*.c"
-        "pq-crypto/sike_r1/fp_generic_r1.c"
-        "pq-crypto/sike_r1/P503_r1.c"
-        "pq-crypto/sike_r1/sike_r1_kem.c"
-        "pq-crypto/sike_r1/fips202_r1.c"
-        "pq-crypto/sike_r2/fips202.c"
-        "pq-crypto/sike_r2/P434.c"
-        )
+if(S2N_NO_PQ)
+    message(STATUS "S2N_NO_PQ flag was detected - Disabling Post Quantum Crypto")
+    # If all Post Quantum Code is disabled, also disable PQ assembly optimized code
+    set(S2N_NO_PQ_ASM ON)
+else()
+    file(GLOB PQ_HEADERS
+        "pq-crypto/*.h"
+        "pq-crypto/bike_r1/*.h"
+        "pq-crypto/bike_r2/*.h"
+        "pq-crypto/sike_r1/*.h"
```

Comment:
what happened to `pq-crypto/sike_r2/*.h`?

Reply:
Fixed